code/game/machinery/_machinery.dm

```diff
@@ -535,6 +535,23 @@ Class Procs:
 		return FALSE
 	return TRUE
 
+// Power cell in hand replacement
+/obj/machinery/attackby(obj/item/C, mob/user)
+	if(istype(C, /obj/item/stock_parts/cell) && panel_open)
+		locate(/obj/item/circuitboard/machine) in component_parts
```

Comment:
You are locating a circuitboard here and then doing nothing with it. Locate returns a value. What was the intent of this code?